[32m2023-05-30 17:40:01 +0200[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 191941ec-e0e1-4b5e-8cae-f7424d66bb8e - 49796 - LOGS_CAPTURED - Started capturing logs in process (pid: 49796).
[32m2023-05-30 17:40:01 +0200[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 191941ec-e0e1-4b5e-8cae-f7424d66bb8e - 49796 - create_delta_table - STEP_START - Started execution of step "create_delta_table".
[32m2023-05-30 17:40:01 +0200[0m - dagster - [34mERROR[0m - [31m__ASSET_JOB - 191941ec-e0e1-4b5e-8cae-f7424d66bb8e - 49796 - create_delta_table - STEP_FAILURE - Execution of step "create_delta_table" failed.

dagster._core.errors.DagsterExecutionLoadInputError: Error occurred while loading input "json_to_flat_properties" of step "create_delta_table"::

AttributeError: 'LocalParquetIOManager' object has no attribute '_get_spark_session'

Stack Trace:
  File "/Users/ctac/Desktop/Data Engineer /Projects/realestate_scraping_project/env/lib/python3.11/site-packages/dagster/_core/execution/plan/utils.py", line 54, in op_execution_error_boundary
    yield
  File "/Users/ctac/Desktop/Data Engineer /Projects/realestate_scraping_project/env/lib/python3.11/site-packages/dagster/_core/execution/plan/inputs.py", line 819, in _load_input_with_input_manager
    value = input_manager.load_input(context)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ctac/Desktop/Data Engineer /Projects/realestate_scraping_project/realestate-scraping/realestate_scraping/__init__.py", line 107, in load_input
    spark = self._get_spark_session
            ^^^^^^^^^^^^^^^^^^^^^^^
[0m
